config/all-in-one/values.yaml

```diff
@@ -81,6 +81,9 @@ cp4waiops:
   # The storage class for large block for CP4WAIOps to use.
   # storageClassLargeBlock: rook-cephfs
 
+  # imageCatalog of cp-shared, override if custom build
+  # imageCatalog: icr.io/cpopen/ibm-operator-catalog:latest
```

Comment:
There is only one imageCatalog covering all that include cp-shared, aimanager, and eventmanager. So, we should use this one.

Reply:
Agree.